examples/stm32/xively_demo_mt/Inc/FreeRTOSConfig.h

```diff
@@ -92,11 +92,11 @@ extern uint32_t SystemCoreClock;
 #define configCPU_CLOCK_HZ ( SystemCoreClock )
 #define configTICK_RATE_HZ ( ( TickType_t )1000 )
 #define configMAX_PRIORITIES ( 7 )
-#define configMINIMAL_STACK_SIZE ( ( uint16_t )128 )
+#define configMINIMAL_STACK_SIZE ( ( uint16_t )256 )
 #if defined( __GNUC__ )
-#define configTOTAL_HEAP_SIZE ( ( size_t )( 32 * 1024 ) )
+#define configTOTAL_HEAP_SIZE ( ( size_t )( 55 * 1024 ) )
 #else
-#define configTOTAL_HEAP_SIZE ( ( size_t )( 32 * 1024 ) )
+#define configTOTAL_HEAP_SIZE ( ( size_t )( 32 * 1024 ) ) off

```

Comment:
Just for future developers that will try to use different toolchain we could change this 32->55 as in case of gnu arm none eabi.